tcms/static/js/testrun_actions.js

```diff
@@ -60,15 +61,30 @@ Nitrate.TestRuns.Details.on_load = function() {
   var toggle_case_run = function(e) {
     var c = jQ(this).parent(); // Container
     var c_container = c.next(); // Content Containers
-    var case_id = c.find('input[name="case"]')[0].value;
-    var case_run_id = c.find('input[name="case_run"]')[0].value;
-    var case_text_version = c.find('input[name="case_text_version"]')[0].value;
-    var type = 'case_run';
+    var caseId = c.find('input[name="case"]')[0].value;
+    var caseRunId = c.find('input[name="case_run"]')[0].value;
+    var caseTextVersion = c.find('input[name="case_text_version"]')[0].value;
     var callback = function(t) {
       // Observe the update case run stauts/comment form
-      c_container.parent().find('.update_form')
-        .unbind('submit').bind('submit', updateCaseRunStatus);
 
+      const caseId = c_container.find('input[name=case_id]').val()
```

Comment:
Remove this line. Value is the same as `caseId` variable coming from line 64. No need to duplicate it.